app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,53 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    policy_contents.each do |content|
+      find_associations!(content)
```

Comment:
base on the above comment, content would be something like content_attributes or something to make it clear it's the attributes we're using to find/create/update a MiqPolicyContent.